marketing_emails/email_kickoff_keynote.md

```diff
@@ -0,0 +1,39 @@
+*Fellow Pythonistas!*
+
+It is my honor to announce that PyTN 2018 is on its way.  The 2018 event will take place on February 10th and 11th at the Nashville Law School.  The organizers are working on setting up the 2018 event to meet the level of excellence former chairman Jason Myers set.
```

Comment:
I'm not sure I want to draw _too much_ attention to the fact that Jason isn't organizing anymore. While that is notable, to the attendee, who's organizing shouldn't make a huge difference. I think I'd make the second sentence about all the things we're doing again this year. Maybe something like:

"Just like previous years, this year's conference will feature keynotes, 3 talk tracks, 1 tutorial track, and a day of Young Coders!"